apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam.env.$envParam.branches/route.tsx

```
@@ -0,0 +1,613 @@
+              <InputGroup className="max-w-full">
+                <Label>Branch name</Label>
+                <Input {...conform.input(branchName)} />
+                <Hint>
+                  Must not contain: spaces <InlineCode variant="extra-small">~</InlineCode>{" "}
+                  <InlineCode variant="extra-small">^</InlineCode>{" "}
+                  <InlineCode variant="extra-small">:</InlineCode>{" "}
+                  <InlineCode variant="extra-small">?</InlineCode>{" "}
+                  <InlineCode variant="extra-small">*</InlineCode>{" "}
+                  <InlineCode variant="extra-small">{"["}</InlineCode>{" "}
+                  <InlineCode variant="extra-small">\</InlineCode>{" "}
+                  <InlineCode variant="extra-small">//</InlineCode>{" "}
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)